src/mmw/package.json

```diff
@@ -28,6 +29,7 @@
     "jstify": "0.9.0",
     "leaflet": "0.7.3",
     "leaflet-draw": "0.2.3",
+    "livereloadify": "2.0.0",
```

Comment:
Ah I see we explicitly specify dependencies in `bundle.sh`. Please ignore the previous comment.